### ISSUE TITLE
[Feature Request] Select multiple videos

### ISSUE THREAD

**Issue by Tunsinasro** [anilbeesetti/nextplayer#688]
Hello and thank you for the good work, Would it be possible to add the ability to select multiple videos to delete them simultaneously?

**Comment by ngocanhtve** [comment]
Delete the entire folder.

**Comment by Tunsinasro** [comment]
@ngocanhtve well it's not possible when you unselect the option to sort videos by folder

**Comment by ngocanhtve** [comment]
Yeah, you're right, deleting the entire folder is useful only in cases where multiple episodes of the same season have been watched.

**Comment by Yashraj254** [comment]
@anilbeesetti can i work on this issue..?

**Comment by anilbeesetti** [comment]
@Yashraj254 Go ahead

**Comment by serrq** [comment]
Multiple selected videos not necessarily is an entire folder.

I was to open a new issue when fortunately I seen this one.

What is the current status of this work?

**Comment by Yashraj254** [comment]
@anilbeesetti does it look good?

https://github.com/anilbeesetti/nextplayer/assets/52420957/b55f1ca9-4a50-487e-90de-9ae0f7fe634f

**Comment by Tunsinasro** [comment]
@Yashraj254 it looks good, thank you

**Comment by VasilisKos** [comment]
That would be really helpful, but not only to delete files, but also to play the selected videos continuously. So, two available options in the bottom pop-up menu: "Play", and "Delete".
Please add this feature.

**Comment by serrq** [comment]
> That would be really helpful, but not only to delete files, but also to play the selected videos continuously. So, two available options in the bottom pop-up menu: "Play", and "Delete". Please add this feature.

This should be another feature: create playlists.

**Comment by VasilisKos** [comment]
> > That would be really helpful, but not only to delete files, but also to play the selected videos continuously. So, two available options in the bottom pop-up menu: "Play", and "Delete". Please add this feature.
> 
> This should be another feature: create playlists.

No, what I'm saying is different than the playlist. It's a quick way to watch some videos, without saving to a playlist file. Maybe that could be a third option in the popup menu: "Add to playlist".

**Comment by Mahantor** [comment]
Please add this feature along with the ability to move and copy videos to other folders and locations in the application.

**Comment by anilbeesetti** [comment]
Completed in 0.15.0 version
